File: com/teamiurg/techexpansion/item/TEItem.java
package com.teamiurg.techexpansion.item;

import net.minecraft.creativetab.CreativeTabs;
import net.minecraft.item.Item;

public class TEItem extends Item {

	public TEItem(String name, CreativeTabs tab, int maxStackSize) {
		this.setUnlocalizedName(name).setCreativeTab(tab).setMaxStackSize(maxStackSize);
	}
}

File: src/main/scala/com/teamisotope/techexpansion/item/TEItem.scala
package com.teamisotope.techexpansion.item

import net.minecraft.creativetab.CreativeTabs
import net.minecraft.item.Item

class TEItem(name: String, tab: CreativeTabs, maxStackSize: Int) extends Item {
  this.setUnlocalizedName(name).setRegistryName(name).setCreativeTab(tab).setMaxStackSize(maxStackSize)
}
